jbpm-system-test/jbpm-springboot-mssql-test/.gitignore

```
@@ -0,0 +1,36 @@
+# Eclipse, Netbeans and IntelliJ files
+/.*
```

forgot to mention, you need to add this to the gitignore file list, since this file is generated by springboot when running the tests
```suggestion
SpringBoot.xml
/.*
```

Also, we should consider the sql script we download from github for the test, otherwise git will consider it as a new file added
https://github.com/kiegroup/jbpm-playground/pull/17/files#diff-be7822dcd332569f1601edfab0cd727cd6341fb3be41ab5e4f3bc9f1e9918372R176